### PR TITLE
Fix issue with podman logs on fresh containers

### LINKED ISSUE #433: podman logs: fails on created (but not started) container
Minor corner case:
```
# podman create --name foo fedora date
7486b62d2316429c94bd547b19f2b809e9905d83c709cc0ba478768548003fe2

# podman logs foo
unable to read container log file: open /var/lib/containers/storage/overlay-containers/7486b62d2316429c94bd547b19f2b809e9905d83c709cc0ba478768548003fe2/userdata/ctr.log: no such file or directory
# echo $?
125
```
(works as expected after `podman start foo`). 

FWIW docker simply emits no output and exits 0, which seems like reasonable behavior to me.

### PR DESCRIPTION
**Pull request by mheon**
Fixes #433

### CODE REVIEW

**Comment by baude** [comment]
lgtm

**Comment by baude** [comment]
@rh-atomic-bot r+

**Comment by mheon** [comment]
@rh-atomic-bot retry

File: cmd/podman/logs.go [hunk 1]
```diff
@@ -104,7 +105,24 @@ func logsCmd(c *cli.Context) error {
...
+	if err != nil {
+		return err
+	}
+
+	// If the log file does not exist yet and the container is in the
+	// Configured state, it has never been started before and no logs exist
+	// Exit cleanly in this case
+	if _, err := os.Stat(logPath); err != nil {
+		if state == libpod.ContainerStateConfigured {
+			logrus.Debugf("Container has not been created, no logs exist yet")
```
Comment (edsantiago) [inline cmd/podman/logs.go]: Day late dollar short, but shouldn't this read 'Container has not been **started**` ?

Comment (mheon) [reply cmd/podman/logs.go]: It's technically correct, as the error will only show up when the container hasn't moved to `ContainerStateCreated`, but agree that your phrasing would make more sense from a user's perspective

**Comment by TomSweeneyRedHat** [comment]
Sorry looks like I've a pending comment similar to @edsantiago.  From a user perspective, I think this error is very confusing.  They've just done 'podman create ...' successfully and now when they do the 'podman log' they're told it's not been created.  I too think we should change that to started.

**Comment by rhatdan** [comment]
Someone open a PR for this. Maybe 
Container logs do not exist, since the container has not be started.

**Pull request closed without merging**